<launch>
  <node pkg="pongbot" name="pongbot_fkin" type="pongbot_fkin"
    output="screen"/>
</launch>